runtime/compiler/control/J9Options.hpp

```diff
@@ -309,6 +309,15 @@ class OMR_EXTENSIBLE Options : public OMR::OptionsConnector
 
    static int32_t _expensiveCompWeight; // weight of a comp request to be considered expensive
    static int32_t _jProfilingEnablementSampleThreshold;
+  
+#if defined(J9VM_OPT_MICROJIT)
+   static int32_t getMJITEnabled() { return _mjitEnabled; }
+   static int32_t _mjitEnabled;
+
+
+   static int32_t getMJITInitialCount() { return _mjitInitialCount; }
+   static int32_t _mjitInitialCount;
```

Comment:
Another TODO for the future: if the microjit counts are not static, then they could be applied per method (through options). In TR_JIT we can do `-Xjit:{myMethod}(count=20)` so it would be nice to be able to do the same for microjit `-Xjit:{myMethod}(microjitcount=10)`